sdk/communication/Azure.Communication.NetworkTraversal/src/CommunicationRelayClientOptions.cs

```diff
@@ -40,6 +41,10 @@ public enum ServiceVersion
             /// The V2021_02_22_preview1 of the networking service.
             /// </summary>
             V2021_02_22_preview1 = 1,
```

Comment:
Should we remove this version? If a developer instantiates the client with this version in the options, what will happen?
Given that the new version is a breaking change and the schema/(de)serializer code has changed, I would expect it to fail.

Reply:
Good point! I guess we'd keep the previous SDK version release but the new version wouldn't include the  `2021_06_21_preview` API version